mesos-master-detector/build.gradle

```diff
@@ -0,0 +1,17 @@
+dependencies {
+    testCompile project(':test-utils')
+    implementation "com.lightbend.akka:akka-stream-alpakka-simple-codecs_$scalaVersion:1.0-M2"
+    implementation "com.pauldijou:jwt-play-json_$scalaVersion:2.1.0"
+    implementation "org.scala-lang.modules:scala-async_$scalaVersion:0.10.0"
+
+    // Transitive dependencies
+    compile project(':persistence-zookeeper')
+    compile "de.heikoseeberger:akka-http-play-json_$scalaVersion:1.26.0"
+    compile "org.apache.mesos:mesos:$mesosVersion"
+}
+
+publishing.publications {
+    mavenJava(MavenPublication) {
+        artifactId 'mesos-master-detector'
```

Comment:
It was there but it would require a dependency on `persistence-zookeeper`. I think `mesos-master-detector` is fine until we have more uses cases. I don't like `util` packages because the often become dustbins for everything.